### ISSUE TITLE
Buggy in Vivaldi Browser

### ISSUE THREAD

**Issue by rkyleg** [andryou/scriptsafe#98]
Has this been tested with the Vivaldi browser? It seems like every time I visit a website for the first time, and click the extension button to allow scripts on the site, it causes the browser to navigate to the previous page. Also, after scripts are allowed, the page doesn't refresh automatically like it does in Chrome. It uses Chromium under the hood.

**Comment by 5teffelchen** [comment]
Same here with ScriptSafe v1.0.8.4 and Vivaldi Snapshot 1.5.604.4. Would be really appreciated to be fixed.

**Comment by andryou** [comment]
Sorry for the late reply, slowly going through all issues. I've tested the latest Vivaldi version (1.6.689.40 (Stable channel) (32-bit)) and it appears this has been fixed.

I know Vivaldi is a rapidly developing browser which had numerous bugs earlier but seems to have matured. The issue you described isn't (wasn't) an issue with ScriptSafe, but rather with Vivaldi.

Side-note: ScriptSafe v1.0.8.5 should be released by end of this week :)

Again, my apologies for the late follow-up.
